tests/test_bakta.nf.test

```diff
@@ -56,8 +56,7 @@ nextflow_pipeline {
                 { assert path("$outputDir/reports/ampcombi2/Ampcombi_summary.tsv").text.contains("KDEMFK_0115") },
                 { assert snapshot(
                     path("$outputDir/reports/ampcombi2/Ampcombi_cluster.log"),
-                    path("$outputDir/reports/ampcombi2/Ampcombi_complete.log"),
-                    path("$outputDir/reports/ampcombi2/Ampcombi_parse_tables.log")
```

Comment:
Why don't you test these two files in a different way?

Reply:
Oh!
- the `comple.log` is checked, just removed comma
- the `pase_table.log` should be tested! Thanks for pointing out. Will fix it.
Context: In the fixes from yesterday it was included twice, so I just removed one. Wasn't properly checking that for these 2 tests.